### PR TITLE
added doc for force flag

### PR DESCRIPTION
**Pull request by unclebconnor**
### What does this PR do?
Added doc for force flag so user can delete an SLO if it exists in a dashboard

### Motivation
Support ticket

### Preview link
Replace the branch name and add the complete path: -->
https://docs-staging.datadoghq.com/bcon/slo-force-delete-api/api/?lang=bash#service-level-objectives

### CODE REVIEW

**Comment by unclebconnor** [comment]
Will add python/ruby as well.  Demoing quick and will come back to complete.

**Comment by unclebconnor** [comment]
@apigirl we're going to just leave this as-is so ok to review

File: content/en/api/service_level_objectives/service_level_objectives_delete.md [hunk 1]
```diff
@@ -11,4 +11,6 @@ Permanently delete a SLO.
 
 **ARGUMENTS**:
 
-This endpoint takes no JSON arguments.
+* **`force`** [*optional*]:
+
+    Boolean: Force delete SLO.  Required if SLO exists in a dashboard.  Default `false`
```
Comment (armcburney) [inline content/en/api/service_level_objectives/service_level_objectives_delete.md]: Could you reword this as: 

> Permanently delete an SLO. 
>
>  If an SLO is used in a dashboard, the `DELETE /v1/slo/` endpoint will return a `409` conflict error because the SLO is referenced in a dashboard.
>
> **ARGUMENTS**:
>
> * **`force`** [*optional*]:
>
> Boolean: Force delete the SLO. The SLO will be deleted regardless of if it's referenced in a dashboard.  Default`false`.

Comment (unclebconnor) [reply content/en/api/service_level_objectives/service_level_objectives_delete.md]: @armcburney nice additions.  Just added 👍

File: content/en/api/service_level_objectives/service_level_objectives_delete.md [hunk 2]
```diff
@@ -9,6 +9,10 @@ external_redirect: /api/#delete-a-service-level-objective
 
 Permanently delete a SLO.
 
+If an SLO is used in a dashboard, the DELETE /v1/slo/ endpoint will return a 409 conflict error because the SLO is referenced in a dashboard.
```
Comment (ruthnaebeck) [inline content/en/api/service_level_objectives/service_level_objectives_delete.md]:
```suggestion
If an SLO is used in a dashboard, the DELETE /v1/slo/ endpoint returns a 409 conflict error because the SLO is referenced in a dashboard.
```

File: content/en/api/service_level_objectives/service_level_objectives_delete.md [hunk 3]
```diff
@@ -9,6 +9,10 @@ external_redirect: /api/#delete-a-service-level-objective
...
+
 **ARGUMENTS**:
 
-This endpoint takes no JSON arguments.
+* **`force`** [*optional*]:
+
+    Boolean: Force delete the SLO. The SLO will be deleted regardless of if it's referenced in a dashboard. Defaultfalse.
```
Comment (ruthnaebeck) [inline content/en/api/service_level_objectives/service_level_objectives_delete.md]:
```suggestion
    Boolean: Force delete the SLO. The SLO is deleted even if it's referenced in a dashboard.
```

File: content/en/api/service_level_objectives/service_level_objectives_delete.md [hunk 4]
```diff
@@ -9,6 +9,10 @@ external_redirect: /api/#delete-a-service-level-objective
 
 Permanently delete a SLO.
 
+If an SLO is used in a dashboard, the DELETE /v1/slo/ endpoint will return a 409 conflict error because the SLO is referenced in a dashboard.
+
 **ARGUMENTS**:
 
-This endpoint takes no JSON arguments.
+* **`force`** [*optional*]:
```
Comment (ruthnaebeck) [inline content/en/api/service_level_objectives/service_level_objectives_delete.md]:
```suggestion
* **`force`** [*optional*, *default*=**False**]:
```
